query_classifier/qc_sqlite/sqlite-src-3110100/src/parse.y

```diff
@@ -307,7 +307,7 @@ ecmd ::= oracle_assignment SEMI.
 explain_kw ::= EXPLAIN.  // Also covers DESCRIBE
 explain_kw ::= DESC.
 
-explain ::= explain_kw tbl_name(A). { pParse->explain = 1; maxscaleExplainTable(pParse, A); }
+explain ::= explain_kw tbl_name(A). { pParse->explain = 1; maxscaleExplainTable(pParse, A); sqlite3SrcListDelete(pParse->db, A); }
```

Comment:
But for some exceptions, the data structures that are passed to the qc_sqlite side are also freed on that side. So please add instead at the end of the maxscaleExplainTable()

    exposed_sqlite3SrcListDelete(pParse->db, pList);

Reply:
Sure. I've tested it and it worked